<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="45796" started-at="2015-12-01T18:09:13Z" finished-at="2015-12-01T18:09:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="45796" started-at="2015-12-01T18:09:13Z" finished-at="2015-12-01T18:09:59Z">
      <class name="com.amp.tests.CustomerAddTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.amp.tests.CustomerAddTest@6a5fc7f7]" name="setup" is-config="true" duration-ms="20007" started-at="2015-12-01T23:39:13Z" finished-at="2015-12-01T23:39:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="customerAddAndCreateMoreCust()[pri:0, instance:com.amp.tests.CustomerAddTest@6a5fc7f7]" name="customerAddAndCreateMoreCust" duration-ms="3066" started-at="2015-12-01T23:39:33Z" finished-at="2015-12-01T23:39:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- customerAddAndCreateMoreCust -->
        <test-method status="FAIL" signature="customerAddAndCreateNewProj()[pri:0, instance:com.amp.tests.CustomerAddTest@6a5fc7f7]" name="customerAddAndCreateNewProj" duration-ms="10153" started-at="2015-12-01T23:39:36Z" finished-at="2015-12-01T23:39:47Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[FAILED TO lOGIN ACTITIMEUnable to locate element: {"method":"name","selector":"username"}
Command duration or timeout: 10.04 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'QSIH-00268', ip: '192.168.0.103', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=42.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: da2a8253-1234-42fa-9cc6-15059b4c85d8
*** Element info: {Using=name, value=username}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: FAILED TO lOGIN ACTITIMEUnable to locate element: {"method":"name","selector":"username"}
Command duration or timeout: 10.04 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'QSIH-00268', ip: '192.168.0.103', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=42.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: da2a8253-1234-42fa-9cc6-15059b4c85d8
*** Element info: {Using=name, value=username}
	at com.amp.pages.Acti_LoginPage.login(Acti_LoginPage.java:110)
	at com.amp.tests.CustomerAddTest.customerAddAndCreateNewProj(CustomerAddTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- customerAddAndCreateNewProj -->
        <test-method status="FAIL" signature="customerAddAndListProj_Cust()[pri:0, instance:com.amp.tests.CustomerAddTest@6a5fc7f7]" name="customerAddAndListProj_Cust" duration-ms="10059" started-at="2015-12-01T23:39:47Z" finished-at="2015-12-01T23:39:57Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[FAILED TO lOGIN ACTITIMEUnable to locate element: {"method":"name","selector":"username"}
Command duration or timeout: 10.06 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'QSIH-00268', ip: '192.168.0.103', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=42.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: da2a8253-1234-42fa-9cc6-15059b4c85d8
*** Element info: {Using=name, value=username}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: FAILED TO lOGIN ACTITIMEUnable to locate element: {"method":"name","selector":"username"}
Command duration or timeout: 10.06 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'QSIH-00268', ip: '192.168.0.103', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=42.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: da2a8253-1234-42fa-9cc6-15059b4c85d8
*** Element info: {Using=name, value=username}
	at com.amp.pages.Acti_LoginPage.login(Acti_LoginPage.java:110)
	at com.amp.tests.CustomerAddTest.customerAddAndListProj_Cust(CustomerAddTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- customerAddAndListProj_Cust -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.amp.tests.CustomerAddTest@6a5fc7f7]" name="tearDown" is-config="true" duration-ms="2411" started-at="2015-12-01T23:39:57Z" finished-at="2015-12-01T23:39:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.amp.tests.CustomerAddTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
